definitions/procedures/restore/regenerate_queues.rb

```diff
@@ -0,0 +1,63 @@
+module Procedures::Restore
+  class RegenerateQueues < ForemanMaintain::Procedure
+    metadata do
+      description 'Regenerate required qpidd and activemq queues while restoring online backup'
+    end
+
+    def ssl_cert
+      "/etc/pki/katello/certs/#{hostname}-qpid-broker.crt"
```

Comment:
Instead of hard-coded strings for `ssl_cert` & `ssl_key`, can't we get these values from any configuration file?

Reply:
+1

Reply:
@johnpmitsch I tried to check with my limited knowledge of qpid and ssl certs, but I cant find exactly which files are keeping all of these certs path which are applicable for qpidd only, if you have any insight on same ?

Reply:
I don't see it specified anywhere with a quick search, others would know this area better than me

@ekohl does this seem ok hardcoded like this or is there a config file that these cert paths are located?

Reply:
I tried to look, but this file is not used anywhere directly. qpid itself stores it in a NSS keystore and that file is more of an intermediate file. It's unlikely to be changed since it'll be dropped in nightly soon when Pulp 2 is going away.

Reply:
Then I think, it would be fine to have hard-coded strings for ssl_cert & ssl_key.

Instead of having three different methods, will it be ok to have one method (example - `qpid_connection_configs`)? and only in case of `amqps_url`, as @johnpmitsch mentioned you can fetch port_number from answers file. 

@upadhyeammit, what do you say?